gateway/src/apicast/policy/echo.lua

```diff
@@ -1,9 +1,22 @@
-local policy = require('apicast.policy')
-local _M = policy.new('Echo Policy')
+local _M = require('apicast.policy').new('Echo Policy')
```

Comment:
Similar to the comment above. This policy is trivial, but thinking about future contributors, I think that a description of 1 or 2 lines would help.